modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/configuration/script_management_configuration.jsp

```diff
@@ -5,4 +5,21 @@
  */
 --%>
 
-<%@ include file="/init.jsp" %>
\ No newline at end of file
+<%@ include file="/init.jsp" %>
+
+<%
+ScriptManagementConfigurationDisplayContext scriptManagementConfigurationDisplayContext = (ScriptManagementConfigurationDisplayContext)request.getAttribute(ScriptManagementConfigurationDisplayContext.class.getName());
```

Comment:
Hey @danvb112, I can see many examples on the portal using `WebKeys` for this purpose. However, there are also several examples using `class.getName()`.

Can you explain to me what motivated the use of `class.getName`? Just in case I need to do this in the future, decide which approach to take.